From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Igor Velkov <325961+iav@users.noreply.github.com>
Date: Thu, 25 Dec 2025 01:30:00 +0000
Subject: [PATCH] exception: include string.h for strlen/strncmp

Fix implicit declarations in cmd/arm/exception64 by pulling in the
standard string helpers.

---
 include/exception.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/include/exception.h b/include/exception.h
index 0d27152ffa..e32eb833af 100644
--- a/include/exception.h
+++ b/include/exception.h
@@ -7,6 +7,7 @@
  * Copyright (c) 2018, Heinrich Schuchardt <xypron.glpk@gmx.de>
  */
 
 #include <command.h>
+#include <string.h>
 
 static int do_exception(struct cmd_tbl *cmdtp, int flag, int argc,
 			char *const argv[])
